x-pack/plugins/security_solution/public/common/components/cell_actions/index.test.tsx

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { render } from '@testing-library/react';
+import { SecurityCellActionsTrigger } from '../../../actions/constants';
+import { CellActionsMode, SecurityCellActions } from '.';
+import { CellActions } from '@kbn/cell-actions';
+
+const MockCellActions = CellActions as jest.Mocked<typeof CellActions>;
+jest.mock('@kbn/cell-actions', () => ({
+  ...jest.requireActual('@kbn/cell-actions'),
+  CellActions: jest.fn(() => <div data-test-subj="cell-actions-component" />),
+}));
+
+const mockGetFieldSpec = jest.fn((_: string) => ({ someFieldSpec: 'theFieldSpec' }));
```

Comment:
NIT:
` { someFieldSpec: 'theFieldSpec' }` is reused in line83, could consider put as a const.

```suggestion
const mockFieldSpec = { someFieldSpec: 'theFieldSpec' };
const mockGetFieldSpec = jest.fn((_: string) => mockFieldSpec);
```